plugins/themes/default/DefaultThemePlugin.inc.php

```diff
@@ -38,19 +31,106 @@ public function isActive() {
 	 * @return null
 	 */
 	public function init() {
-		// Load Noto Sans font from Google Font CDN
+
+		// Register theme options
+		$this->addOption('typography', 'radio', array(
+			'label' => 'plugins.themes.default.option.typography.label',
+			'description' => 'plugins.themes.default.option.typography.description',
+			'options' => array(
+				'notoSans' => 'plugins.themes.default.option.typography.notoSans',
+				'notoSerif' => 'plugins.themes.default.option.typography.notoSerif',
+				'notoSerif_notoSans' => 'plugins.themes.default.option.typography.notoSerif_notoSans',
+				'notoSans_notoSerif' => 'plugins.themes.default.option.typography.notoSans_notoSerif',
+				'lato' => 'plugins.themes.default.option.typography.lato',
+				'lora' => 'plugins.themes.default.option.typography.lora',
+				'lora_openSans' => 'plugins.themes.default.option.typography.lora_openSans',
+			)
+		));
+
+		$this->addOption('baseColor', 'color', array(
+			'label' => 'plugins.themes.default.option.color.label',
+			'description' => 'plugins.themes.default.option.color.description',
+			'default' => '#1E6292',
+		));
+
+		// Load primary stylesheet
+		$this->addStyle('stylesheet', 'styles/index.less');
+
+		// Store additional LESS variables to process based on options
+		$additionalLessVariables = '';
+
+		// Load fonts from Google Font CDN
 		// To load extended latin or other character sets, see:
 		// https://www.google.com/fonts#UsePlace:use/Collection:Noto+Sans
 		if (Config::getVar('general', 'enable_cdn')) {
-			$this->addStyle(
-				'fontNotoSans',
-				'//fonts.googleapis.com/css?family=Noto+Sans:400,400italic,700,700italic',
-				array('baseUrl' => '')
-			);
+
+			if ($this->getOption('typography') === 'notoSerif') {
+				$this->addStyle(
+					'fontNotoSerif',
+					'//fonts.googleapis.com/css?family=Noto+Serif:400,400i,700,700i',
+					array('baseUrl' => '')
```

Comment:
If no `baseUrl` is defined, the URL to the theme directory will be prepended, so adding a style is [simple like this](https://github.com/NateWr/default-child#add-a-less-file).

You can pass any `baseUrl` if you wanted (eg - for another theme or plugin or whatever). But an empty `baseUrl` string just tells it to prepend nothing.